chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -8,6 +8,10 @@
         <delete-triggers-in-group>triggerJob</delete-triggers-in-group>
         <delete-triggers-in-group>interruptJob</delete-triggers-in-group>
         <!-- BEGIN REMOVE after PROD push -->
+        <delete-job>
```

Comment:
There were some loggers added as part of the initial PR. Those ought to be removed in this PR as well, right?